js/lesson4/tutorial.md

```diff
@@ -104,7 +104,7 @@ The following code allows you to listen for a keypress on the input field, and t
 
 ```js
 $(document).ready(function(){
-  $(document).on('keypress', '#username', function(e){
+  $(document).on('keypress', '#username', function(){
```

Comment:
Ah, maybe. I just saw a student type it last night and assumed it was a typo, so we changed it before debugging it.